typescript/ws-protocol-examples/src/examples/publish-client.ts

```diff
@@ -0,0 +1,70 @@
+import { MessageWriter as Ros1MessageWriter } from "@foxglove/rosmsg-serialization";
+import { FoxgloveClient } from "@foxglove/ws-protocol";
+import { Command, Option } from "commander";
+import Debug from "debug";
+import WebSocket from "ws";
+
+const log = Debug("foxglove:simple-client");
+Debug.enable("foxglove:*");
+
+let running = true;
+
+process.on("SIGINT", () => {
+  running = false;
+});
+
+async function main(url: string, format: "json" | "ros1") {
```

Comment:
The type for `format` is not correct. When logging it, I get `{ format: 'ros1' }`

```suggestion
async function main(url: string, args: { format: "json" | "ros1" }) {
```